extensions/amp-date-display/amp-date-display.md

```diff
@@ -193,3 +194,9 @@ date to UTC.
 
 The `offset-seconds` attribute specifies an integer number of seconds to shift
 the given date.
+
+### data-options-\* (optional)
+
+The `data-options-*` supports all the options under [Intl.DateTimeFormat.options](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat#parameters)
+parameter that specifies the formatting style to use for `localeString` format.
+Valid attributes include: `data-options-date-style`, `data-options-time-style`, etc.
```

Comment:
> suggestion: Let's include a comprehensive list so supported options are explicit in the data-options- format.

Actually we didn't include the supported options explicitly in the code, all parameters with `data-options-` prefix are fetched and passed to `DateTimeFormat`. Thus would prefer keeping as-is (attached the link) because the param settings should be changed along with the original doc. WDYT?

---
> It's also worth calling out that timeZone is specifically unsupported because it is always overwritten with UTC

Good idea, added a note about timeZone conversion when `display-in` is set to `utc`.

Reply:
Will resolve the comment, feel free to open if there's any concern.